pkg/virt-operator/resource/apply/delete.go

```diff
@@ -461,3 +449,151 @@ func DeleteAll(kv *v1.KubeVirt,
 	}
 	return nil
 }
+
+func crdInstanceDeletionCompleted(crd *extv1.CustomResourceDefinition) bool {
+	// Below is an example of what is being looked for here.
+	// The CRD will have this condition once a CRD which is being
+	// deleted has all instances removed related to this CRD.
+	//
+	//    message: removed all instances
+	//    reason: InstanceDeletionCompleted
+	//    status: "False"
+	//    type: Terminating
+
+	if crd.DeletionTimestamp == nil {
+		return false
+	}
+
+	for _, condition := range crd.Status.Conditions {
+		if condition.Type == extv1.Terminating &&
+			condition.Status == extv1.ConditionFalse &&
+			condition.Reason == "InstanceDeletionCompleted" {
```

Comment:
question: who's responsible of updating this condition? Is it something being done by Kuberentes?

Reply:
yes

Reply:
thanks